channels/email/email-headers.rst

```diff
@@ -0,0 +1,160 @@
+
+Ticket attributes
+-----------------
+
+Zammad allows you to use headers to manipulate ticket creations or follow ups.
+The manipulation can be used instead of triggers. Triggers are considered
+*after* header settings and thus can still overrule.
+
+   .. note:: **🔎 Zammad differentiates in between ticket creation and follow up**
+
+      | For creations use: ``X-Zammad-Ticket-{Attribute Name}``
+      | For follow ups use: ``X-Zammad-Ticket-FollowUp-{Attribute Name}``
+
+      This allows you to ensure the changes are only applied in the
+      required situation.
+
+   .. warning:: **🧐 About values**
+
+      * While headers are case insensitive, values like e.g. priority names
+        are case censitive: ``1 low`` will work, but ``1 lOw`` will not!
+      * When using attributes that require datetime values, ensure to use
+        Time Zoned Times. e.g. for 28th September 2021 on 8 am CEST, either use:
+
+            * ``2021-09-28T08:00:00+0200``
+            * ``2021-09-28T08:00:00+02:00``
+            * ``2021-09-28T06:00:00.000Z``
+
+``X-Zammad-Ticket-Priority`` & ``X-Zammad-Ticket-FollowUp-Priority``
+   | Allows you to adjust a tickets priority.
+   | Example: ``X-Zammad-Ticket-Priority: 1 low``
+
+``X-Zammad-Ticket-Group`` & ``X-Zammad-Ticket-FollowUp-Group``
+   | Allows you interfere with regular channel routing of the ticket.
+   | Example: ``X-Zammad-Ticket-Group: Sales``
+
+``X-Zammad-Ticket-Owner`` & ``X-Zammad-Ticket-FollowUp-Owner``
+   | Directly assign or change the ticket owner. Valid values are either
+     ``login`` or ``email``
+   | Example: ``X-Zammad-Ticket-Owner: jdoe``
+
+``X-Zammad-Ticket-State`` & ``X-Zammad-Ticket-FollowUp-State``
+   | Set a specific ticket state.
+   | Example: ``X-Zammad-Ticket-State: closed``
+
+      .. note::
+
+         | Pending states always require the ``pending_time`` attribute on top. 
+         | Like so: 
+           ``X-Zammad-Ticket-Pending_Time: 2021-09-26T08:00:00+0200``
+
+``X-Zammad-Customer-Email``
+   | Manipulate the ticket customer - this can be a different user than the
+     actual sender. Replying to the original sender is still possible.
+   | Example: ``X-Zammad-Customer-Email: jdoe@example.com``
+
+      .. note::
+
+         This header is not available for follow ups.
+
+``X-Zammad-Customer-Login``
+   | Manipulate the ticket customer - this can be a different user than the
+     actual sender. Replying to the original sender is still possible.
+   | Example: ``X-Zammad-Customer-Login: jdoe``
+
+      .. note::
+
+         This header is not available for follow ups.
+
+Article attributes
+------------------
+
+If needed Zammad allows you to manipulate attributes or states of fetched
+email articles.
+
+``X-Zammad-Article-Sender``
+   | Manipulate the sender type (Agent, Customer or System)
+   | Example: ``X-Zammad-Article-Sender: System``
+
+      .. warning::
+
+         System emails are indicated like trigger response like entries.
```

Comment:
would rephrase this sentence, maybe: "System emails are indicated in a similar way as trigger-response-like entries"

Reply:
That's a much better wording, thansk! 🙌